node/pkg/governor/mainnet_chains.go

```diff
@@ -10,21 +10,22 @@ import (
-		chainConfigEntry{emitterChainID: vaa.ChainIDBSC, dailyLimit: 5_000_000, bigTransactionSize: 500_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDPolygon, dailyLimit: 5_000_000, bigTransactionSize: 500_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDAvalanche, dailyLimit: 5_000_000, bigTransactionSize: 500_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDOasis, dailyLimit: 500_000, bigTransactionSize: 50_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDAlgorand, dailyLimit: 200_000, bigTransactionSize: 20_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDAurora, dailyLimit: 500_000, bigTransactionSize: 50_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDFantom, dailyLimit: 500_000, bigTransactionSize: 50_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDKarura, dailyLimit: 500_000, bigTransactionSize: 50_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDAcala, dailyLimit: 500_000, bigTransactionSize: 50_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDKlaytn, dailyLimit: 1_000_000, bigTransactionSize: 100_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDCelo, dailyLimit: 1_000_000, bigTransactionSize: 100_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDNear, dailyLimit: 200_000, bigTransactionSize: 20_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDTerra2, dailyLimit: 500_000, bigTransactionSize: 50_000},
+		{emitterChainID: vaa.ChainIDSolana, dailyLimit: 50_000_000, bigTransactionSize: 5_000_000},
+		{emitterChainID: vaa.ChainIDEthereum, dailyLimit: 50_000_000, bigTransactionSize: 5_000_000},
+		{emitterChainID: vaa.ChainIDTerra, dailyLimit: 1_000_000, bigTransactionSize: 100_000},
+		{emitterChainID: vaa.ChainIDBSC, dailyLimit: 5_000_000, bigTransactionSize: 500_000},
+		{emitterChainID: vaa.ChainIDPolygon, dailyLimit: 5_000_000, bigTransactionSize: 500_000},
+		{emitterChainID: vaa.ChainIDAvalanche, dailyLimit: 5_000_000, bigTransactionSize: 500_000},
+		{emitterChainID: vaa.ChainIDOasis, dailyLimit: 500_000, bigTransactionSize: 50_000},
+		{emitterChainID: vaa.ChainIDAlgorand, dailyLimit: 200_000, bigTransactionSize: 20_000},
+		{emitterChainID: vaa.ChainIDAurora, dailyLimit: 500_000, bigTransactionSize: 50_000},
+		{emitterChainID: vaa.ChainIDFantom, dailyLimit: 500_000, bigTransactionSize: 50_000},
+		{emitterChainID: vaa.ChainIDKarura, dailyLimit: 500_000, bigTransactionSize: 50_000},
+		{emitterChainID: vaa.ChainIDAcala, dailyLimit: 500_000, bigTransactionSize: 50_000},
+		{emitterChainID: vaa.ChainIDKlaytn, dailyLimit: 1_000_000, bigTransactionSize: 100_000},
+		{emitterChainID: vaa.ChainIDCelo, dailyLimit: 1_000_000, bigTransactionSize: 100_000},
+		{emitterChainID: vaa.ChainIDNear, dailyLimit: 200_000, bigTransactionSize: 20_000},
+		{emitterChainID: vaa.ChainIDTerra2, dailyLimit: 500_000, bigTransactionSize: 50_000},
+		{emitterChainID: vaa.ChainIDAptos, dailyLimit: 50_000, bigTransactionSize: 5_000},
```

Comment:
This is going to break things if aptos is not in mainnet! If the governor fails to find aptos in KnownTokenbridgeEmitters, it throws an error on init.

Reply:
Flipped it to draft as a precaution.

Reply:
Would also suggest considering higher limits.